lib/http-bridge/test/integration/Test/Integration/Scenario/CLISpec.hs

```diff
@@ -150,3 +153,9 @@ specWithCluster = do
         [ "whisper", "control", "diary", "solid", "cattle", "salmon"
         , "whale", "slender", "spread", "ice", "shock", "solve"
         , "panel", "caution", "upon", "scatter", "broken", "tonight"]
+
+-- | Run a command using the 'cardano-wallet' executable. We run it through
+-- stack as we intend to also get code-coverage from running these commands!
+cardanoWallet :: CmdResult r => [String] -> IO r
+cardanoWallet args =
+    command [] "stack" (["exec", "--", "cardano-wallet"] ++ args)
```

Comment:
Cool, shouldn't be hard to get it work with what I did https://github.com/input-output-hk/cardano-wallet/pull/278/files#diff-77505846ff3a93c2de96a9af0279e12aR469 ;)

Probably could be also employed here -> https://github.com/input-output-hk/cardano-wallet/pull/278/files#diff-77505846ff3a93c2de96a9af0279e12aR478